integration-tests/tests/register-user.js

```diff
@@ -1,13 +1,13 @@
 const { withUser } = require('../utils/integration-helpers')
-const { fillIn, findText, click, clickable, input, signIntoTerra, dismissNotifications } = require('../utils/integration-utils')
+const { fillIn, findText, click, clickable, input, signIntoTerra } = require('../utils/integration-utils')
+const { fillInReplace } = require('../utils/integration-utils')
 
 
 const testRegisterUserFn = withUser(async ({ page, testUrl, token }) => {
-  await page.goto(testUrl)
+  await page.goto(testUrl, { waitUntil: ['load', 'domcontentloaded', 'networkidle0'], timeout: 60 * 1000 })
```

Comment:
How would you feel about factoring out 

```
{ waitUntil: ['load', 'domcontentloaded', 'networkidle0'], timeout: 60 * 1000 }
```

by adding it into `integration-utils` with something like

```javascript
const waitUntilLoadedOrTimeout = timeout => ({ waitUntil: ['load', 'domcontentloaded', 'networkidle0'], timeout })
```

?

Reply:
I agree that it would be nice to have a method for this-- it appears in a few different files.